master/buildbot/interfaces.py

```diff
@@ -129,6 +129,236 @@ def getAddress(user):
         be reached by email), or a Deferred which will fire with the same."""
 
 
+class ITestResult(Interface):
```

Comment:
Having test case oriented stats is a useful thing.
But like you said on ml there are too much unfinished stuff in buildbot.